packages/fiori/src/ShellBar.js

```diff
@@ -464,14 +466,24 @@ class ShellBar extends UI5Element {
 		};
 
 		this._handleResize = async event => {
-			await this._getResponsivePopover();
-			this.overflowPopover.close();
-			this._overflowActions();
+			this._debounce(async function() {
```

Comment:
Do not use a normal function, unless needed. Arrow function preferred:

`this._debounce(async () => {
......
}, HANDLE_RESIZE_)`